tests/Fixer/FunctionNotation/PhpdocToParamTypeFixerTest.php

```diff
@@ -0,0 +1,290 @@
+                    * @param string $bar
+                    * @param int $baz
+                    * @param float $tab
+                    */
+                    function my_foo(string $bar, int $baz, float $tab) {}',
+                '<?php
+                    /**
+                    * @param string $bar
+                    * @param int $baz
+                    * @param float $tab
+                    */
+                    function my_foo($bar, $baz, $tab) {}',
+            ],
+            'non-root class with mixed type of param' => [
+                '<?php
+                    /**
+                    * @param mixed $bar
+                    */
+                    function my_foo($bar) {}',
+            ],
+            'non-root namespaced class' => [
+                '<?php /** @param My\Bar $foo */ function my_foo(My\Bar $foo) {}',
+                '<?php /** @param My\Bar $foo */ function my_foo($foo) {}',
+            ],
+            'root class' => [
+                '<?php /** @param \My\Bar $foo */ function my_foo(\My\Bar $foo) {}',
+                '<?php /** @param \My\Bar $foo */ function my_foo($foo) {}',
+            ],
+            'interface' => [
+                '<?php interface Foo { /** @param Bar $bar */ function my_foo(Bar $bar); }',
+                '<?php interface Foo { /** @param Bar $bar */ function my_foo($bar); }',
+            ],
+            'invalid void param on ^7.1' => [
+                '<?php /** @param null|void $foo */ function my_foo($foo) {}',
+            ],
+            'iterable return on ^7.1' => [
+                '<?php /** @param iterable $counter */ function my_foo(iterable $counter) {}',
+                '<?php /** @param iterable $counter */ function my_foo($counter) {}',
+                70100,
+            ],
+            'array native type' => [
+                '<?php /** @param array $foo */ function my_foo(array $foo) {}',
+                '<?php /** @param array $foo */ function my_foo($foo) {}',
+            ],
+            'callable type' => [
+                '<?php /** @param callable $foo */ function my_foo(callable $foo) {}',
+                '<?php /** @param callable $foo */ function my_foo($foo) {}',
+            ],
+            'self accessor' => [
+                '<?php
+                    class Foo {
+                        /** @param self $foo */ function my_foo(self $foo) {}
+                    }
+                ',
+                '<?php
+                    class Foo {
+                        /** @param self $foo */ function my_foo($foo) {}
+                    }
+                ',
+            ],
+            'static is skipped' => [
+                '<?php
+                    class Foo {
+                        /** @param static $foo */ function my_foo($foo) {}
+                    }
+                ',
+            ],
+            'skip resource special type' => [
+                '<?php /** @param resource */ function my_foo() {}',
```

Comment:
Shouldn't you have the params, to be sure it's skipped?
This case isn't doing anything anyway now because of doc/signature mismatch.

Same for the following 3.

Reply:
yeah, you are right I fix this